src/example3.cpp

```diff
@@ -81,7 +81,7 @@ int main(int /* argc */, char ** /* argv */) {
 
     int width, height;
     glfwGetFramebufferSize(window, &width, &height);
-    glViewport(0, 0, width, height);
```

Comment:
It is called on every frame in Screen::drawWidgets. The glViewport call in the initialization code seems unneccessary. Actually, the two lines above
```
int width, height;
glfwGetFramebufferSize(window, &width, &height);
```
can be deleted as well, without visible side-effects. These two can of course also be left in to demonstrate the API usage.